.circleci/config.yml

```diff
@@ -61,17 +61,3 @@ workflows:
       - e2eTestChartInstall:
           requires:
             - build
-
-      - deploy:
-          filters:
-            branches:
-              only: master
-          requires:
-            - e2eTestChartInstall
```

Comment:
`deploy` currently errors because this is the workflow but there is no deploy job.